jenkins/e2e-image/e2e-runner.sh

```diff
@@ -432,6 +432,12 @@ elif [[ ! "${JOB_NAME}" =~ -pull- ]]; then
   fi
 fi
 
+if [[ -n "${PRIORITY_PATH:-}" ]]; then
+  export PATH="${PRIORITY_PATH}:${PATH}"
+fi
+echo "kubectl on path is:"
+kubectl version --client
```

Comment:
@spxtr you're probably thinking of https://github.com/kubernetes/kubernetes/pull/37049, which is only tangentially related, needs work, and is unclear when I'll have time to look at again.